Added flexget.subfolder.conf.sample

XxAcielxX opened the pull request:
flexget.subfolder.conf.sample for Swag

<!--- Provide a general summary of your changes in the Title above -->

[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]
	

<!--- Before submitting a pull request please check the following -->

<!---  That you have made a branch in your fork, we'd rather not merge from your master -->
<!---  That if the PR is addressing an existing issue include, closes #<issue number> , in the body of the PR commit message   -->
<!---  You have included links to any files / patches etc your PR may be using in the body of the PR commit message -->
<!---  -->

##  Thanks, team linuxserver.io

STaRDoGG commented:
Oops, guess I should've checked the PR's before creating another one. The one I just added also includes the API as well.

XxAcielxX commented:
> Oops, guess I should've checked the PR's before creating another one. The one I just added also includes the API as well.

True, if you have checked before, you could have added the API block to my commited file. No worries, I have added it in mine as well now.

Roxedus reviewed (approved):
LGTM
